### ISSUE TITLE
Cannot copy executable

### ISSUE THREAD

**Issue by tduccuong** [libgdx/packr#119]
hi, im targeting linux64 and got this error:
```
Cleaning output directory '/path/to/targets/linux64' ...
Copying executable ...
java.io.FileNotFoundException: targets/linux64/testapp (No such file or directory)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
	at com.badlogicgames.packr.Packr.copyExecutableAndClasspath(Packr.java:142)
	at com.badlogicgames.packr.Packr.pack(Packr.java:50)
	at com.badlogicgames.packr.Packr.main(Packr.java:343)
```
I am sure (checked and rechecked) that I created the `targets/linux64` before running packr.

thanks for helping!

**Comment by tduccuong** [comment]
since no one responded, I fixed the problem myself. See [this](https://github.com/libgdx/packr/pull/120) if someone else faces same problem as I did.

**Comment by smelc** [comment]
Thanks for the fix, I had the same problem and your commit fixed it.

**Comment by code-disaster** [comment]
Fix merged in #120.
